src/web/EmployerFavouritesWeb/Domain/ReadModel/ApprenticeshipFavourites.cs

```diff
@@ -12,8 +12,8 @@ public WriteModel.ApprenticeshipFavourites MapToWriteModel()
             model.AddRange(this.Select(x => new Domain.WriteModel.ApprenticeshipFavourite
             {
                 ApprenticeshipId = x.ApprenticeshipId,
-                Ukprns = x?.Providers.Select(y => y.Ukprn).ToList() ?? new List<int>(0)
-            }));
+                Providers = x?.Providers.Select(y => new WriteModel.Provider(y.Ukprn, y.Locations )).ToList()  // ?? new List<Provider>()
```

Comment:
We need the commented out code don't we for a default?

Reply:
Sorted